src/main/java/reposense/git/GitCloner.java

```diff
@@ -19,6 +21,9 @@ public static void downloadRepo(String organization, String repoName, String bra
             throw new GitClonerException(e);
             //Due to an unsolved bug on Windows Git, for some repository, Git Clone will return an error even
             // though the repo is cloned properly.
+        } catch (IOException ioe) {
+            System.out.println("Error creating/deleting directory " + organization + File.separator + repoName
```

Comment:
I think here should throw `GitClonerException` because it can't proceed anymore without downloading the repo.

Reply:
Would prefer not to print the message here, since its throwing to the higher level to handle the error.